src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    add(result, a) {
+      return result + a;
+    },
```

Comment:
1. you should work with one function parameter
2. use this

```suggestion
    add(value) {
      this.result += value;
      return this;
    },
```